src/ckb/channel.rs

```diff
@@ -1873,6 +1867,8 @@ impl ChannelActorState {
                     "Setting local_committed_at for tlc {:?} to commitment number {:?}",
                     tlc.tlc.id, commitment_numbers
                 );
+                // FIXME: need to confirm with @contrun
+                #[allow(clippy::if_same_then_else)]
                 if is_received {
                     tlc.creation_confirmed_at = Some(commitment_numbers);
                 } else {
```

Comment:
@contrun please confirm this, is this unintentionally?

Reply:
The is_received check should be deleted.